From cef129408e2be315f65f81d0e97f879099fe7e41 Mon Sep 17 00:00:00 2001
From: Hans Wennborg <hans@chromium.org>
Date: Tue, 11 Jun 2024 15:21:21 +0000
Subject: [PATCH] build_rust.py: Final typo fix.

Bug: 343232416
Change-Id: I7b83bf24b7d996297c09d9412ea26618cbefdda4
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5621932
Commit-Queue: danakj <danakj@chromium.org>
Auto-Submit: Hans Wennborg <hans@chromium.org>
Commit-Queue: Hans Wennborg <hans@chromium.org>
Reviewed-by: danakj <danakj@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1313425}
---
 tools/rust/build_rust.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/rust/build_rust.py b/tools/rust/build_rust.py
index cd18ef5fb942a7..d246d6b3b50a8e 100755
--- a/tools/rust/build_rust.py
+++ b/tools/rust/build_rust.py
@@ -803,8 +803,8 @@ def main():
     # Delete the build directory.
     if not args.skip_clean:
         print('Clearing build directory...')
-        if os.path.exists(BUILD_DIR):
-            RmTree(BUILD_DIR)
+        if os.path.exists(RUST_BUILD_DIR):
+            RmTree(RUST_BUILD_DIR)
 
     if not args.skip_test:
         print(f'Building stage 2 artifacts and running tests...')
